lib/support/clone_vm.rb

```diff
@@ -395,15 +395,34 @@ def full_clone?
       options[:clone_type] == :full
     end
 
+    def root_folder
+      vim.serviceInstance.content.rootFolder
```

Comment:
Should this be ```@root_folder ||= vim.serviceInstance.content.rootFolder``` so we avoid the lookup cost?

Reply:
I didn't do this initially out of concern for what I didn't know about the Vmami API. Considering the uses of it though, it is probably safe and makes sense.